commands.go

```diff
@@ -537,6 +538,23 @@ func (c cmdable) Command(ctx context.Context) *CommandsInfoCmd {
 	return cmd
 }
 
+func (c cmdable) CommandList(ctx context.Context, filter *FilterBy) *StringSliceCmd {
+	args := []interface{}{"command", "list"}
```

Comment:
recommend: args := make([]interface{}, 0, 5)

It is a good habit for us to apply for the required memory capacity at one time to avoid memory copying in slice expansion.

Reply:
Understood, Thanks @monkey92t 
Sorry for noob mistake